### PR TITLE
Processing chain: added init-api.R file to chain scripts

### LINKED ISSUE #636: add init-api.R to processing chain

### PR DESCRIPTION
**Pull request by SteRiccio**
Resolves #636

### CODE REVIEW

File: server/modules/analysis/service/_rChain/_files/system/init-api.R [hunk 1]
```diff
@@ -0,0 +1,39 @@
+usePackage("httr");
```
Comment (minotogna) [inline server/modules/analysis/service/_rChain/_files/system/init-api.R]: add this to init.r (or even better, let's create a init-packages.R where all the needed packages will go. there are a lot of them )

File: server/modules/analysis/service/_rChain/_files/system/init-api.R [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  resp <-httr::POST(
+    paste(arena.host, 'auth/login', sep = ''), 
+    body = list(email=user, password=password)
+  )
+  respParsed <- arena.parseResponse(resp)
+  if("message" %in% names(respParsed) && respParsed$message == 'validationErrors.user.userNotFound'){
+    stop( respParsed$message )
+  }
+  print(paste('User', user,'succesfully logged in', sep = ' '))
+}
```
Comment (minotogna) [inline server/modules/analysis/service/_rChain/_files/system/init-api.R]: add new lines among functions

File: server/modules/analysis/service/_rChain/_files/system/init-api.R [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+arena.login()
```
Comment (minotogna) [inline server/modules/analysis/service/_rChain/_files/system/init-api.R]: this must be the first line of arena.R

Comment (SteRiccio) [reply server/modules/analysis/service/_rChain/_files/system/init-api.R]: ok

File: server/modules/analysis/service/_rChain/_files/system/init-api.R (hunk 2 above)
Comment (SteRiccio) [reply server/modules/analysis/service/_rChain/_files/system/init-api.R]: ok

File: server/modules/analysis/service/_rChain/_files/system/init-api.R (hunk 1 above)
Comment (SteRiccio) [reply server/modules/analysis/service/_rChain/_files/system/init-api.R]: ok

**Pull request merged**